test/ctia/bundle/core_test.clj

```diff
@@ -44,3 +48,33 @@
             :query "source_ref:*malware*"}
            (sut/relationships-filters "id" {:source_type :malware
                                             :related_to [:source_ref]})))))
+
+(deftest with-existing-entity-test
+  (testing "with-existing-entity"
+    (let [indicator-id-1 (make-id "indicator")
+          indicator-id-2 (make-id "indicator")
+          indicator-id-3 (make-id "indicator")]
+      (testing "1 existing external id"
```

Comment:
You could try an `are` here

As I see it, it's not only about DRYing, but also about making the differences evident (rn I'd have to squint and look for differences very carefully)

Reply:
Tried to address this with a test fn, it's a little bit "harder" to make it readable with logging test. Tell me what you think.